tailcfg/tailcfg.go

```diff
@@ -926,6 +926,14 @@ type PingRequest struct {
 	// Log is whether to log about this ping in the success case.
 	// For failure cases, the client will log regardless.
 	Log bool `json:",omitempty"`
+
```

Comment:
these new docs don't explain the relationship between the URL-only are-you-there pings and the pings with a type+IP.

Reply:
updated comments at the top of PingRequest